docs/docs/cmd/spo/list/list-get.md

```diff
@@ -29,18 +32,30 @@ m365 spo list get [options]
 
 ## Examples
 
-Return information about a list with ID _0cd891ef-afce-4e55-b836-fce03286cccf_ located in site _https://contoso.sharepoint.com/sites/project-x_
+Get information about a list with ID _0cd891ef-afce-4e55-b836-fce03286cccf_ located in site _https://contoso.sharepoint.com/sites/project-x_
```

Comment:
Along the way could we correct the approach in spec no to duplicate the data that is in the example and just use a more general explanation 👍
Like: 
```suggestion
Get information about a list with specified ID located in the specified site
```